content/en/security_platform/cspm/_index.md

```diff
@@ -0,0 +1,60 @@
+
+Assess the configuration of your cloud resources, such as security groups, storage buckets, load balancers, and databases against CSPM controls. Use the Datadog Agent to review local configuration information from servers, containers, and Kubernetes clusters against security best practices.
+
+View your cloud security posture at a higher level level with the CSPM Home page, and drill into the details of findings and analyze historical configurations with the Findings Explorer.
+
+## Posture Management
+
+View your security posture with respect to Datadog’s OOTB security posture rules at a higher level with the [Home page][4], with links to [findings][5] in more detail.
+
+{{< img src="security_platform/cspm/getting_started/posture-management-home.png" alt="Homepage for CSPM">}}
+
+## Glossary
+
+- **Average security configuration score**: Percentage of your environment that satisfies all of your active [Datadog’s OOTB rules][6]. Formula: `(# of resources with 0 findings) / (total # of resources scanned)`.
+
+- **Requirement**:  A group of controls representing a single technical or operational topic, such as _Access Management_ or _Networking_. The regulatory framework PCI DSS, for example, has [12 requirements][7].
+
+- **Control**: A specific recommendation for how technology, people, and processes should be managed; typically based on a regulation or industry standard.
+
+- **Resource**: A configurable entity that needs to be continuously scanned for adherence with one or more controls. Examples of AWS instance resources include hosts, containers, security groups, users, and customer-managed IAM policies.
+
+{{< img src="security_platform/cspm/getting_started/resource.png" alt="CSPM resource" style="width:65%;">}}
+
+- **Rule**: A rule evaluates the configuration of a resource to validate a particular security best practice. These rules partially or completely, measure adherence to controls. Rules map to multiple controls, requirements, and frameworks.
+
+{{< img src="security_platform/cspm/getting_started/rules.png" alt="Rules and frameworks" style="width:100%;">}}
+
+- **Framework**: A collection of requirements that map to an industry benchmark or regulatory standard.
+
+{{< img src="security_platform/cspm/getting_started/frameworks.png" alt="Frameworks overview" style="width:100%;">}}
+
+## Get Started
+
+{{< whatsnext >}}
+  {{< nextlink href="/security_platform/cspm/getting_started">}}Configure your cloud environment for CSPM scanning.{{< /nextlink >}}
+  {{< nextlink href="/security_platform/cspm/findings">}} Search and explore findings to reveal the details about a misconfigured resource, enabling you to identify the steps needed to fix it.{{< /nextlink >}}
+  {{< nextlink href="/security_platform/cspm/frameworks_and_benchmarks">}}Get details about Datadog’s OOTB configuration rules to compare your configuration to applicable regulatory standards, controls, and industry benchmarks.{{< /nextlink >}}
+{{< /whatsnext >}}
```

Comment:
The numbers for the links on this page are showing. Looks like it might be because you didn't list them down here.

Reply:
Smooth catch, never pushed my last commit. Resolved now!